multinode-demo/stop_nodes.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+ip_addr_file=$1
+remote_user=$2
+
+usage() {
+  echo -e "\\tUsage: $0 <IP Address array> <username> [path to ssh keys]\\n"
+  echo -e "\\t <IP Address array>: A bash script that exports an array of IP addresses, ip_addr_array. Elements of the array are public IP address of remote nodes."
+  echo -e "\\t <username>:         The username for logging into remote nodes."
+  exit 1
+}
+
+# Sample IP Address array file contents
+# ip_addr_array=(192.168.1.1 192.168.1.5 192.168.2.2)
+
+if [[ -z "$ip_addr_file" ]]; then
```

Comment:
I like using a one liner for this kind of thing: `[[ -n $ip_addr_file ]] || usage`